website/docs/remote-control/commands.md

```diff
@@ -0,0 +1,52 @@
+---
+title: MQTT and HTTP commands
+---
+
+Interact and control the application and device remotely using either MQTT or HTTP (REST) commands, including using your device as an announcer with Google Text-To-Speach.
```

Comment:
Typos from the old wiki (e.g. "speach" instead of "speech" here) are still intact for now. I just reformatted some of the Markdown in some cases to comply with markdownlint.

Reply:
That's fine. We could later do a proof-check and then do everything in one sweep